stock_warehouse_flow/models/stock_warehouse_flow.py

```diff
@@ -0,0 +1,367 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.safe_eval import safe_eval
+
+logger = logging.getLogger(__name__)
+
+
+class StockWarehouseFlow(models.Model):
+    _name = "stock.warehouse.flow"
+    _description = "Stock Warehouse Routing Flow"
+    _order = "sequence"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    sequence = fields.Integer(string="Sequence", default=10)
+    warehouse_id = fields.Many2one(
+        comodel_name="stock.warehouse",
+        ondelete="restrict",
+        string="Warehouse",
+        default=lambda o: o._default_warehouse_id(),
+        required=True,
+    )
+    from_picking_type_id = fields.Many2one(
+        comodel_name="stock.picking.type",
+        ondelete="restrict",
+        string="From operation type",
+        required=True,
+        domain="[('code', '=', 'outgoing')]",
```

Comment:
you said above we support incoming too, seems not the case here, no?

Reply:
`index=True`?

Reply:
Incoming is not supported yet (we do not need it), but this is something the module could support later so I mention this in the ROADMAP file.